java/basic/src/main/java/io/sentry/example/basic/MyClass.java

```diff
@@ -0,0 +1,109 @@
+package io.sentry.example.basic;
+
+import io.sentry.Sentry;
+import io.sentry.SentryClient;
+import io.sentry.SentryClientFactory;
+import io.sentry.context.Context;
+import io.sentry.event.BreadcrumbBuilder;
+import io.sentry.event.UserBuilder;
+
+public class MyClass {
+    private static SentryClient sentry;
+
+    public static void main(String... args) {
+        /*
+        It is recommended that you use the DSN detection system, which
+        will check the environment variable "SENTRY_DSN", the Java
+        System Property "sentry.dsn", or the "sentry.properties" file
+        in your classpath. This makes it easier to provide and adjust
+        your DSN without needing to change your code. See the configuration
+        page for more information.
+        */
+        Sentry.init();
+
+        // You can also manually provide the DSN to the ``init`` method.
+        // String dsn = "https://<SENTRY_PUBLIC_KEY>:<SENTRY_PRIVATE_KEY>@sentry.io/<PROJECT_ID>";
+        // Sentry.init(dsn);
+
+        /*
+        It is possible to go around the static ``Sentry`` API, which means
+        you are responsible for making the SentryClient instance available
+        to your code.
+        */
+        sentry = SentryClientFactory.sentryClient();
+
+        MyClass myClass = new MyClass();
+        myClass.logWithStaticAPI();
+        myClass.logWithInstanceAPI();
```

Comment:
I'd actually remove this call and the instance class. It's an example in the documentation that 99% of users shouldn't use IMO and it may be confusing.